package.json

```diff
@@ -23,14 +23,14 @@
     "url": "https://github.com/apostrophecms/eslint-config-apostrophe/issues"
   },
   "homepage": "https://github.com/apostrophecms/eslint-config-apostrophe#readme",
-  "peerDependencies": {
-    "eslint": ">= 8.39.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-n": "^16.0.0",
-    "eslint-plugin-promise": "^6.1.1"
-  },
   "dependencies": {
-    "eslint-config-standard": "^17.1.0"
+    "eslint": "^8.53.0",
```

Comment:
we should keep `>=` for projects wanting to use eslint 9+

Reply:
got back